docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment
+sidebar_label: Controller
+id: deploy
+---
+
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md';
+
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
+
+## Installation
+
+The controller package `openziti-controller` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.

```

Comment:
"cross-platform install script" somehow rings a bit strange for me. I usually think of cross-platform as being able to run on more than a single platform (linux). I know what you're shooting for here, but in the spirit of "less is more", it feels like this could just remove "cross-platform". Also I'd like to remove "easiest". This guide should just stick to the facts:
You can install the `openziti` package by executing the convenience script or by installing the package manually. something like that